ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-ref.yaml

```diff
@@ -84,6 +84,12 @@ ref:
   - name: CLUSTER_TIMEOUT
     default: "7200"
     documentation: Set to number of seconds for the cluster to timeout if it's not ready.
-
+  - name: HYPERSHIFT_NODE_COUNT
```

Comment:
Why are we editing the default vars for this ref here?

I believe this is what is bringing in all of the extra tests that we saw from running the /pj-rehearse command.

Just a reminder we shouldn't run just /pj-rehearse always specify the exact job name when you run /pj-rehearse so that you don't accidentally run other teams tests.

Reply:
I would say these three are not used in the "rosa" workflow. You might need to set `REPLICAS` to 4. The variable REPLICAS seems to be used in the firewatch-rosa-aws-sts-hypershift workflow. Then you probably don't need  HYPERSHIFT_BASE_DOMAIN and BASE_DOMAIN either.